### PR TITLE
Finished navbar and homepage routing

### PR DESCRIPTION
**Pull request by xgmak94**
Navbar backend done
Homepage done

### CODE REVIEW

File: server/routes.js [hunk 1]
```diff
@@ -32,6 +30,29 @@ router.get('/country/:countryid', (req, res) => {
...
+      }
+    })
+    .catch((err) => res.sendStatus(404));
+});
+
+router.post('/user/balance', (req, res) => {
+  addBalance(req.body)
+    .then((results) => res.status(201).send(results))
+    .catch((err) => res.sendStatus(404));
+
```
Comment (andychow94) [inline server/routes.js:56]: Missing closing brackets - }); on line 55

Comment (xgmak94) [reply server/routes.js:56]: Fixed

**Comment by andychow94** [comment]
I noticed the roulette overlaps the balance modal, but otherwise the code looks good! Gonna approve

**Review by andychow94 — APPROVED** [review]
I noticed the roulette overlaps the balance modal, but otherwise the code looks good! Gonna approve

**Review by Jesse132 — APPROVED** [review]
Looks good. Approved for merge.

**Pull request merged**